src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -237,7 +237,7 @@ trait BeamHelper extends LazyLogging {
     }
     assert(
       !isConfigArgRequired || (isConfigArgRequired && parsedArgs.config.isDefined),
-      "config is a required value, and must yield a valid config."
+      "Beam config is a required, must provide a valid configuration file."
```

Comment:
Can you change to "..., please provide a valid..." or "..., you must provide..." or ", a valid configuration file must be provided.".

And in fact, while we are improving it, let's help a bit more and add an example:

> Beam config is required.... For example:
>  ./gradlew :run -PappArgs="['--config', 'test/input/beamville/beam.conf']"